web/business-registry-dashboard/app/utils/account.ts

```diff
@@ -76,8 +76,8 @@ export async function addBusinessRegistryDashboardSubscription (orgId: number |
  */
 export async function hasActiveBusinessRegistryDashboardSubscription (orgId: number | string): Promise<boolean> {
   try {
-    const { $authApi } = useNuxtApp()
-    const products = await $authApi<string>(`/orgs/${orgId}/products`)
+    const { $authApiBRD } = useNuxtApp()
```

Comment:
This couldn't be done once at the top of this module?

Reply:
Ah, I see -- there isn't a class in this module and that's why it looks different.

However, the function you created definitely cleaned up the code 👍